tests/data/TMT16_ursgal_lookup.csv

```diff
@@ -0,0 +1,4 @@
+lineage_root,file,spectrum_id,ms_level,rt,rt_unit,preset_scan_configuration,total_ion_current,ion_injection_time,ion_injection_time_unit,filter_string,scan_window_lower_limit,scan_window_upper_limit,precursor_mz,precursor_charge,precursor_id,resolution,ms2_isolation_width,ms2_isolation_mz,gex_id,faims_cv
```

Comment:
lineage_root is a ursgal concept and should not be used / imposed / confuse people outside of ursgal. Maybe rename it (back?) to raw_data_location